launch/index.ts

```diff
@@ -0,0 +1,78 @@
+import { options } from "@parallel-finance/api";
+import { ApiPromise, Keyring, WsProvider } from "@polkadot/api";
+import { exit } from "process";
+
+function sleep(ms: number) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+async function main() {
+  const api = await ApiPromise.create(
+    options({
+      provider: new WsProvider("ws://localhost:9947"),
+    })
+  );
+
+  const chainHeight = async () => {
+    const {
+      block: {
+        header: { number: height },
+      },
+    } = await api.rpc.chain.getBlock();
+    return height.toNumber();
+  };
+
+  do await sleep(1000);
+  while ((await chainHeight()) === 0);
+
+  const keyring = new Keyring({ type: "sr25519", ss58Format: 110 });
+  const signer = keyring.addFromUri("//Dave");
+
+  let call: any[] = [];
```

Comment:
can we remove this any if possible

Reply:
Of course it should, It's better in environment variable